src/test/lib/storage/pos_lists/entire_chunk_pos_list_test.cpp

```diff
@@ -61,4 +61,37 @@ TEST_F(EntireChunkPosListTest, AddAfterMatchedAllTest) {
   // TODO(XPERIANER): Maybe add a better check than just size, cause the returned iterators should also handle
   // this case, which we right now don't check.
 }
+
+TEST_F(EntireChunkPosListTest, CheckIterators) {
+  auto table = Table::create_dummy_table({{"a", DataType::Int, false}});
+  table->append({int32_t{1}});
+  table->append({int32_t{2}});
+  table->append({int32_t{3}});
+
+  EXPECT_EQ(table->chunk_count(), 1);
+  EXPECT_EQ(table->row_count(), 3);
+
+  const auto chunk_id = ChunkID{0};
+  const auto chunk_size = ChunkOffset{3};
```

Comment:
Why assign to own variables? I would be happier if I saw an explanation of the values in some comments.